### DISCUSSION TITLE
How to explain where to put files in a module using Sampler's-scaffolding

### DISCUSSION THREAD

**Discussion by johlju** [gaelcolas/Sampler#414] [Ideas]
If a module is created and contain for example a public command. The module is published. Then there is a need for another command that needs a private function, or a class-based resource. There is a lot of options where to place prefix code, public command, private command, enums, classes, MOF-based resources, class-based resource.

How shall we document where it is recommended to place files in the folder structure so the module is built the correct way?

**Comment by gaelcolas** [comment]
Have a look at what Jaykul wrote, and we need to expand: https://github.com/PoshCode/ModuleBuilder#organizing-your-module
